php-checks/src/test/resources/checks/PHPDeprecatedFunctionUsageCheck.php

```diff
@@ -156,4 +156,16 @@ function f() {
 
 use Rubix\ML\Graph\Nodes\Split;
 
-new Split();   // Ok
+new Split();   // OK
+?>
+
+<?php
+namespace mynamespace;
+
+function split() {
+  echo "split";
+}
+
+split();  // OK, function in current namespace
```

Comment:
Seems like an issue of the test construction and not of the logic, but when I evaluate `((FunctionCallTreeImpl) tree).symbol().qualifiedName().toString()`(from `ifFunctionRedefinedInCurrentNamespace`) it would produce `rubix\ml\graph\nodes\split`. Something is not correct in this construction. I would create another test file to test the namespace individually.